national-resources/estonia.md

```diff
@@ -0,0 +1,121 @@
+
+## Introduction 
+This page gathers COVID-19 disease data initiatives, dashboards, and datasets that were developed and published during 2020-2024 in Estonia. It also provides information on health authorities and local regulations for data handling and disease control. The primary target audience is the wider scientific community.
+
+## Health authorities
+<!--- A section to list and provide context to agencies/authorities/institutions which define public health measures and policies --->
+In Estonia, infectious disease data is mainly collected by [The Estonian Health Board](https://www.terviseamet.ee/en) and made publicly available as open data. The Health Board also provides guidelines:
+* for the health authorities (all guidelines in Estonian)
+  * [Monkeypox guidelines for healthcare workers](https://www.terviseamet.ee/et/ahvirougete-juhised-tervishoiutootajatele)
+  * [Hospital infections](https://www.terviseamet.ee/et/nakkushaigused-menuu/tervishoiutootajale/haiglanakkus)
+* and the general public
+  * [Monkeypox guidelines](https://www.terviseamet.ee/et/ahvirouged)
+  * [COVID-19](https://www.terviseamet.ee/et/nakkus-haigused/covid-19-infoleht)
+
+Since 2021, the Health Board has provided weekly [epidemic reviews](https://www.terviseamet.ee/et/koroonaviirus/epiidulevaated) on SARS-CoV-2 and flu situations that capture the infection rates, prevalent strains, hospitalisation, and death statistics. 
+
+The Health Board provides further information regarding infectious diseases on its home page, such as [safety guidelines and recommendations](https://www.terviseamet.ee/et/nakkushaigused-menuu/tervishoiutootajale/nakkusohutuse-juhendid-ja-soovitused) or [interactive dashboard](https://www.terviseamet.ee/et/nakkus-haigused/tervishoiutootajale/nakkushaigustesse-haigestumine) for the infectious disease history in Estonia but the majority of the information is available only in Estonian.
+
+
+## Disease surveillance initiatives
+During the COVID-19 pandemic, the need for additional surveillance data became pressing. With scientists being able to ensure rapid development and implementation of new methodologies, the following initiatives were implemented in cooperation with the University of Tartu:
+
+* [__Wastewater surveillance__](https://ut.ee/en/node/113141) was set up in August 2020 [by the researchers of the University of Tartu](https://ut.ee/en/node/113141) (prof Tanel Tenson and Veljo Kisand) and taken over [by the Health Board](https://www.terviseamet.ee/et/reoveeseire-kaardirakendus) in 2022. Based on data from Estonia, it was observed that the viral load in wastewater increased before a rise in the number of infected people was observed. Furthermore, in certain places such as dormitories and prisons, the viral load increase was used to predict outbreaks. In conclusion, wastewater studies had an additional value when infection rates were on the rise or decline, but not during a plateau.
```

Comment:
Maybe remove titles like "Prof". We probably don't know them all, and they change anyway.